src/main/resources/com/cloudbees/hudson/plugins/folder/views/Messages.properties

```
@@ -0,0 +1 @@
+Hudson.ViewName=All
```

```suggestion
Hudson.ViewName=All
```

(trailing newlines please)